swap/peer.go

```diff
@@ -221,3 +230,19 @@ func (p *Peer) sendCheque() error {
 		Cheque: cheque,
 	})
 }
+
+// getBouncedCheque retrieves if a bounced cheque exists at address from the blockchain
+func (p *Peer) getBouncedCheque() (bool, error) {
```

Comment:
the function is called `getBoucedCheque`, so it should either change its name, or return a cheque instead of a boolean